src/components/va-accordion/va-accordion-item.css

```diff
@@ -20,7 +25,7 @@ button {
   background-position: right 2rem center;
   background-repeat: no-repeat;
   background-size: 1.5rem;
-  display: inline-block;
+  display: block;
```

Comment:
This is the fix to the first issue - everything else is about adding a dynamic header.